pages/pipelines/example_pipelines.md

```diff
@@ -1,7 +1,8 @@
 # Example pipelines
 
-A list of repositories containing example [pipelines](/docs/pipelines).
+Example [pipelines](/docs/pipelines) can be found in our [pipeline template gallery](https://buildkite.com/pipelines/templates) covering a wide range of technologies (such as [javascript](https://buildkite.com/pipelines/templates?language=javascript)) and use cases (such as [infrastructure as code](https://buildkite.com/pipelines/templates?useCase=iac)).
```

Comment:
```suggestion
Example [pipelines](/docs/pipelines) can be found in our [pipeline template gallery](https://buildkite.com/pipelines/templates) covering a wide range of technologies (such as [JavaScript](https://buildkite.com/pipelines/templates?language=javascript)) and use cases (such as [infrastructure as code](https://buildkite.com/pipelines/templates?useCase=iac)).
```

Reply:
Thanks very much for your update @clbarrell !

This minor correction should hopefully stop the linter from breaking...